src/electeeManagement/views.py

```diff
@@ -28,9 +28,42 @@ def all_electees(request):
     requirements = dict(
         (requirements.requirement, requirements) for requirements in Requirements.objects.all())
 
+    # process the electee progress, so that wo don't have to run those if-else in template html
+    print type(electee_list)
+    electee_list_plain = []
+    for electee in electee_list:
+        progress = {}
+        progress['uniqname'] = electee.member.uniqname
+        progress['first_name'] = electee.member.first_name
+        progress['last_name'] = electee.member.last_name
+        is_undergraduate = electee.member.is_undergraduate()
+
+        progress['num_socials_approved'] = electee.num_socials_approved
+        progress['num_service_hours_approved'] = electee.num_service_hours_approved
+        progress['electee_interview'] = electee.electee_interview
+        progress['electee_exam'] = electee.electee_exam
+        progress['dues'] = electee.dues
```

Comment:
This seems unnecessarily verbose. Why create an empty dictionary and then set every value?  Why not just do something like 
`progress = { 'uniqname' : electee.member.uniqname, 'first_name' : electee.member.first_name ...}` similar to how context is created on 65. If you really love the assignment feel, you can also do  `{ uniqname=electee.member.uniqname, first_name=electee.member.first_name ...}`

If you're feeling even fancier, you can probably do something like http://stackoverflow.com/questions/9058305/getting-attributes-of-a-class since you're basically just translating the member name into a dictionary.